scripts/link-validator.conf

```diff
@@ -43,101 +43,32 @@ site-link-validator {
   ignore-missing-local-files-regex = ""
   // e.g. "^api/alpakka/snapshot/pekko/stream/alpakka/googlecloud/storage/impl/Formats.*"
 
+  ignore-files = [
+    # This file is generated, we can't do much about the invalid links here
+    "project/license-report.html"
```

Comment:
Seems ok to me

Reply:
see more context: https://github.com/apache/pekko/pull/704

Reply:
I'm not sure if I understand the context correctly. It seems the license report was disabled because the report generation is slow, not the link validation. So before we remove that logic, we should fix https://github.com/sbt/sbt-license-report/issues/87 ?

Reply:
My bad, I thought the ISSUE could also solve the problem of report speed.